src/Microsoft.VisualStudio.Editor.UnitTests/AuthoringTests.txt

```diff
@@ -0,0 +1,131 @@
+﻿==========================================================================
+    Visual Studio Team System: Overview of Authoring and Running Tests    
```

Comment:
With this change, I am bringing in all the files. This is just a port of the projects itself. There will be a separate pass(which will take some time) to clean all of these files and remove unwanted Razzle variables. 

This change will build a test binary containing nothing in it.